### PR TITLE
Update 0.11

### PR DESCRIPTION
**Pull request by afreof**
Is the mailing list for meta-efibootguard still the way to go?  Or are PR acceptable?

### CODE REVIEW

**Comment by jan-kiszka** [comment]
Unless I miss something, you are not yet packaging the new linux-stub and the related tool (needed for unified kernel images / secure boot).

**Comment by hoinmic** [comment]
Thank you for the commit. For me a PR is ok. 
I think it would be an advantage if we could leave the license currently on "GPLv2". If someone is not yet on the latest Yocto version. I meant there is only a warning displayed.

**Comment by afreof** [comment]
> I think it would be an advantage if we could leave the license currently on "GPLv2".

You are right, the SPDX license files are not available on the dunfell branch. They have been added with [hardknott](https://git.yoctoproject.org/poky/tree/meta/files/common-licenses/GPL-2.0-only?h=hardknott)

Maybe we should create a dunfell branch which stays with efibootguard 0.10 and remove dunfell from LAYERSERIES_COMPAT on the master branch. Considering dunfell as a stable bugfix only branch would also have some advantages.

**Comment by afreof** [comment]
> you are not yet packaging the new linux-stub and the related tool

You are right, this needs to be handled better. The last push splits these new features into separate optional packages.

My understanding is that bg_gen_unified_kernel is supposed to run on the build machine and does not necessarily end up on the target device. Therefore I split it into a separate package.

Also the new kernel-stub is going to an optional package which is used on the target device for secure boot.

**Comment by jan-kiszka** [comment]
Right, `bg_gen_unified_kernel` is not needed on a normal embedded device, nor is the stub if you generate the unified kernel image (which contains the stub) only during imaging.

**Comment by hoinmic** [comment]
The branch "dunfell" and "dunfell_next" for Yocto LTS Dunfell are now created.
https://github.com/siemens/meta-efibootguard/tree/dunfell
https://github.com/siemens/meta-efibootguard/tree/dunfell_next

**Comment by hoinmic** [comment]
The PR looks good. I'll take it on the next branch right away. Thank you.

File: conf/layer.conf [hunk 1]
```diff
@@ -17,4 +17,4 @@ BBFILE_COLLECTIONS += "efibootguard"
 BBFILE_PATTERN_efibootguard = "^${LAYERDIR}/"
 BBFILE_PRIORITY_efibootguard = "6"
 
-LAYERSERIES_COMPAT_efibootguard = "dunfell hardknott honister kirkstone"
+LAYERSERIES_COMPAT_efibootguard = "hardknott honister kirkstone"
```
Comment (hoinmic) [inline conf/layer.conf:20]: I will update the readme in a moment and include this change.

**Pull request merged**